packages/core/documentation/Core/transformers.md

```diff
@@ -1,17 +1,18 @@
 # Transformers
 
-A transformer converts the message `format` from the backend to the Hyral and the other way around.
+A transformer converts the message *format* from the backend to the Hyral and the other way around.
```

Comment:
this reads a bit funnily. "Converts to the Hyral". Converts to the hyral format?